### PR TITLE
website: Add iPerf3 tutorial

### PR DESCRIPTION
**Pull request by kfirtoledo**
Add iperf3 tutorials to documentation.

### CODE REVIEW

File: website/content/en/docs/tutorials/iperf.md [hunk 1]
```diff
@@ -1,12 +1,228 @@
...
+    ```
+
+    Server cluster:
+
+    ```sh
+    export PEER1_IP=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' peer1-control-plane`
+    envsubst < $IPERF3_FILES/clusterlink/peer1.yaml | kubectl apply -f -
+    ```
+
+    Note: The `PEER_IP` refers to the node IP of the peer Kind Cluster, which assigns the peer YAML file
```
Comment (elevran) [inline website/content/en/docs/tutorials/iperf.md:164]: consider, on the entire document, if using the notice/alert etc shortcodes would be helpful.
You can see examples in peers.md file

File: website/content/en/docs/tutorials/iperf.md [hunk 2]
```diff
@@ -1,12 +1,228 @@
...
+    kubectl apply -f $IPERF3_FILES/clusterlink/allow-policy.yaml
+    ```
+
+    Server cluster:
+
+    ```sh
+    kubectl apply -f $IPERF3_FILES/clusterlink/allow-policy.yaml
+    ```
+
+    For more details about policy see [ClusterLink policies](https://github.com/clusterlink-net/clusterlink/docs/concepts/policies.md)
```
Comment (elevran) [inline website/content/en/docs/tutorials/iperf.md:196]:
```suggestion
    For more details about policy see [ClusterLink policies](https://clusterlink.net/docs/concepts/policies)
```

**Review by elevran — CHANGES_REQUESTED** [review]
please fix the URLs - everything else is up to your discretion

File: website/content/en/docs/tutorials/iperf.md [hunk 3]
```diff
@@ -1,12 +1,228 @@
 ---
-title: iperf
+title: iPerf3 Tutorial
```
[the remark this answers is not in the document]
Comment (kfirtoledo) [reply website/content/en/docs/tutorials/iperf.md:2]: In the official website is iPerf
https://iperf.fr/iperf-download.php

File: website/content/en/docs/tutorials/iperf.md [hunk 4]
```diff
@@ -1,12 +1,228 @@
...
+    kind create cluster --name=peer2
+    ```
+
+1. Setup `KUBECONFIG` on each terminal to access the cluster:
+
+    Client cluster:
+
+    ```sh
+    kubectl config use-context kind-peer1
+    cp ~/.kube/config $PWD/config-peer1
```
[the remark this answers is not in the document]
Comment (kfirtoledo) [reply website/content/en/docs/tutorials/iperf.md:62]: I added it

File: website/content/en/docs/tutorials/iperf.md [hunk 5]
```diff
@@ -1,12 +1,228 @@
...
+    ```
+
+    For more details about fabric and peer concepts see [ClusterLink Concepts](https://github.com/clusterlink-net/clusterlink/docs/concepts/)
+
+1. Deploy ClusterLink on each cluster:
+
+    Client cluster:
+
+    ```sh
+    clusterlink deploy peer --name peer1 --autostart --ingress=NodePort --ingress-port=30443
```
[the remark this answers is not in the document]
Comment (kfirtoledo) [reply website/content/en/docs/tutorials/iperf.md:118]: Yes it start the CRD

File: demos/iperf3/testdata/manifests/clusterlink/peer1.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+apiVersion: clusterlink.net/v1alpha1
+kind: Peer
+metadata:
+  name: peer1
+  namespace: clusterlink-system
+spec:
+  gateways:
+    - host: "${PEER1_IP}"
+      port: 30443
```
[the remark this answers is not in the document]
Comment (kfirtoledo) [reply demos/iperf3/testdata/manifests/clusterlink/peer1.yaml:9]: Yes

File: website/content/en/docs/tutorials/iperf.md [hunk 7]
```diff
@@ -1,12 +1,228 @@
...
+## Enable Access between the clusters
+
+In this step, we enable connectivity access between the iPerf3 client and server.
+
+1. First, we add the peers to each cluster:
+
+    Client cluster:
+
+    ```sh
+    export PEER2_IP=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' peer2-control-plane`
```
[the remark this answers is not in the document]
Comment (kfirtoledo) [reply website/content/en/docs/tutorials/iperf.md:153]: I prefer the hardcoding

File: website/content/en/docs/tutorials/iperf.md [hunk 8]
```diff
@@ -30,48 +29,50 @@ Before you start, you must have access to two Kubernetes clusters.
...
+    kind create cluster --name=client
     ```
 
     Server cluster:
 
     ```sh
     cd iperf3-tutorial
-    kind create cluster --name=peer2
+    kind create cluster --name=server
     ```
```
Comment (elevran) [inline website/content/en/docs/tutorials/iperf.md:54]: nit: for users not too familiar with kind, consider adding a note/tip that kind names clusters' context in the kubeconfig with prefix `kind-` so name would be different than specified on the command line?

**Review by elevran — APPROVED** [review]

File: website/content/en/docs/tutorials/iperf.md [hunk 9]
```diff
@@ -1,12 +1,233 @@
...
+    Server cluster:
+
+    ```sh
+    kubectl config use-context kind-server
+    cp ~/.kube/config $PWD/config-server
+    export KUBECONFIG=$PWD/config-server
+    ```
+
+    {{< notice info >}}
+    You can run the tutorial in a single terminal and switch access between the clusters using  `kubectl config use-context kind-peerX`.
```
[the remark this answers is not in the document]
Comment (kfirtoledo) [reply website/content/en/docs/tutorials/iperf.md:74]: I did it the same way  as the sites/peers page https://deploy-preview-448--clusterlink-net.netlify.app/docs/concepts/sites/

File: demos/iperf3/testdata/manifests/clusterlink/allow-policy.yaml [hunk 10]
```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: clusterlink-system
 spec:
   action: allow
-  privileged: true
+  privileged: false
```
[the remark this answers is not in the document]
Comment (kfirtoledo) [reply demos/iperf3/testdata/manifests/clusterlink/allow-policy.yaml:8]: I will wait for it to merge and fix it.

File: website/content/en/docs/tutorials/iperf.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (kfirtoledo) [reply website/content/en/docs/tutorials/iperf.md:74]: I change it to note instead of info.

**Pull request merged**